client/app/components/project-geography.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import EmberObject, { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const bblObject = EmberObject.extend({
+  dcp_validatedlot: '',
+  dcp_bblnumber: '',
+  versionnumber: '',
+  dcp_partiallot: null,
+  dcp_developmentsite: null,
+});
+
+export default class ProjectGeographyComponent extends Component {
+  // object that mimicks the objects that exist in the bbls array
+  // of the pasForm (expand entity: dcp_dcp_projectbbl_dcp_pasform)
+  bblObject = bblObject.create();
```

Comment:
I would make this a [POJO](https://en.wikipedia.org/wiki/Plain_old_Java_object)... Im not sure EmberObject is giving us anything here. I _do_ like that you're helping clarify the expected structure of the object upfront :)